libraries/standard/mqtt/src/iot_mqtt_api.c

```diff
@@ -1148,6 +1148,20 @@ IotMqttError_t IotMqtt_Connect( const IotMqttNetworkInfo_t * pNetworkInfo,
 
         if( pNewMqttConnection != NULL )
         {
+            /* Coverity checker USE_AFTER_FREE found a warning when trying to
```

Comment:
Should mention that this is a false positive early in these comments.

Reply:
Fixed it.